### PR TITLE
add doc about degree 3 folding

### PR DESCRIPTION
**Pull request by fabrizio-m**

### CODE REVIEW

**Review by volhovm — CHANGES_REQUESTED** [review]
@fabrizio-m could you please add all the equations to the doc here? Like how we compute these `t_0` and `t_1` especially.

**Comment by dannywillems** [comment]
@fabrizio-m: we can discuss about [this](https://github.com/o1-labs/proof-systems/pull/2243#pullrequestreview-2065856409) tomorrow at the offsite. I am happy to guide you on this. We should go through the codebase and the equations together.

File: folding/src/lib.rs [hunk 1]
```diff
@@ -20,7 +20,37 @@
 //! expressions in orther to support multiple constraints.
 //! That results in 2 errors terms t_0 and t_1 and a slightly different computation
```
Comment (querolita) [inline folding/src/lib.rs:21]: results in 2 error terms

**Comment by dannywillems** [comment]
Most of the work in https://github.com/o1-labs/proof-systems/pull/2259 has been done by Fabrizio, I was guided by his explanation and he answered my question. I would suggest closing this one, and move on with 2259.

**Pull request closed without merging**